README.md

```diff
@@ -149,12 +149,14 @@
 - [Gauzy](https://github.com/ever-co/gauzy) - Open-Source Profits Sharing Platform for modern agencies and studios.
 - [LXDhub](https://github.com/Roche/lxdhub) - Management system for Linux Containers (LXC).
 - [Notadd](https://github.com/notadd/notadd) - Microservice development architecture.
+- [Novu](https://github.com/novuhq/novu) - The open-source notification infrastructure with fully functional embedded notification center.
```

Comment:
Do you think it would make more sense to link to the nest package in the repo? I didn't see any immediate links in the README and want to make sure people who navigate here are aware of what they're looking at and how it is related to Nest

Reply:
The title of the section suggests this is a project built with NestJS, right? My intention was to add this project to the list because their API (and some other parts) is built with NestJS - important part of the project. Look at other projects in this section - for example ToolJet has server built with NestJS and this is reason why is listed here I guess.

But I understand that in case of larger monorepo it can be confusing which part is built with NestJS.

Reply:
Ah, nope, that's totally my bad there. I thought this was being added as an integration package, not as a project. The README preview in the diff can be hard to get a grasp of where it is sometimes :laughing: thanks for the correction. Where you have it is perfectly fine as is